indico/web/templates/forms/_form.html

```diff
@@ -9,13 +9,14 @@
 {%- endmacro %}
 
 {% macro form_header(form, method='post', action='', orientation='horizontal', id=none, classes='', multipart=false,
-                     i_form=true, no_ajax=false) -%}
+                     i_form=true, no_ajax=false, ajax_form=false) -%}
```

Comment:
I fully agree with @ThiefMaster. I had overlooked the fact that the caller is passed in the attributed section.